texar/torch/modules/embedders/embedders.py

```diff
@@ -258,8 +258,10 @@ def num_embeddings(self) -> int:
 
     @property
     def output_size(self) -> int:
-        r"""The feature size of :meth:`forward` output :attr:`outputs`.
-        Same as embedding dimension's last element or itself.
+        r"""The feature size of :meth:`forward` output. If embedding
+        dimension is a ``list`` or ``tuple``, the feature size is
+        same as its last element. Otherwise if embedding dimension
+        is a ``int``, the feature size is same as itself.
```

Comment:
```
The feature size of :meth:`forward` output. If the :attr:`dim`
hyperparameter is a ``list`` or ``tuple``, the feature size
equals its final dimension; otherwise, if :attr:`dim` is an
``int``, the feature size equals :attr:`dim`.
```